### PR TITLE
stats are shown on the admin

### LINKED ISSUE #2065: Display user stats on admin `/manage_user/[id]` page
Display a statistic section like on `/account` page.

### PR DESCRIPTION
**Pull request by vivasvan1**
Closes #2065

This is my first commit to OS so please guide me in case i make mistake. Appreciate all feedback. 

image

### CODE REVIEW

File: website/public/locales/en/leaderboard.json [hunk 1]
```diff
@@ -35,5 +35,6 @@
   "weekly": "Weekly",
   "xp_progress_message": "You need {{need, number, integer}} more points to reach the next level!",
   "your_account": "Your account",
-  "your_stats": "Your statistics"
+  "your_stats": "Your statistics",
+  "user_stats": "User statistics"
```
Comment (notmd) [inline website/public/locales/en/leaderboard.json:39]: You dont need to translate text in the admin area

File: website/src/pages/api/user_stats.ts [hunk 2]
```diff
@@ -3,7 +3,14 @@ import { createApiClientFromUser } from "src/lib/oasst_client_factory";
 import { getBackendUserCore } from "src/lib/users";
 
 const handler = withoutRole("banned", async (req, res, token) => {
-  const user = await getBackendUserCore(token.sub);
+  let userid: string;
+  if (req.query.uid) {
```
Comment (notmd) [inline website/src/pages/api/user_stats.ts:7]: Also can you check the current user has any mod or admin role herer? We dont want to expose stats to every one.

Comment (vivasvan1) [reply website/src/pages/api/user_stats.ts]: done

File: website/public/locales/en/leaderboard.json (hunk 1 above)
Comment (vivasvan1) [reply website/public/locales/en/leaderboard.json]: removed it

**Review by notmd — APPROVED** [review]
Thank you!

**Comment by vivasvan1** [comment]
@notmd thanks 🙏 happy to contribute.

**Pull request merged**